modules/jdbc-test/src/test/java/org/testcontainers/jdbc/JDBCDriverTest.java

```diff
@@ -78,9 +81,9 @@ public void test() throws SQLException {
         }
     }
 
-    private void performSimpleTest(String jdbcUrl) throws SQLException {
+    private void performSimpleTest(String jdbcUrl, boolean pmdKnownBroken) throws SQLException {
```

Comment:
I'm not so happy with adding an additional param to the method if it's only needed by on tested class.
The tests will fail if not set?

Reply:
Method **getParameterMetaData** not implemented on clickhouse jdbc driver.